kubernetes/manifests/dragonfly/mainframe/ingress.yaml

```
@@ -7,6 +7,11 @@ metadata:
   name: dragonfly-ingress
   annotations:
     cert-manager.io/cluster-issuer: letsencrypt
+    nginx.ingress.kubernetes.io/server-snippet: |
+      location ~* /metrics {
```

`suggestion(non-blocking)`: We don't really need regexp matching here.
```suggestion
      location /metrics {
```